workspaces/marketplace/plugins/marketplace-backend/src/services/MarketplaceService.ts

```diff
@@ -21,5 +21,8 @@ import {
 
 export interface MarketplaceService {
   getPlugins(): Promise<MarketplacePluginEntry[]>;
+  getPluginByName(name: string): Promise<MarketplacePluginEntry>;
   getPluginList(): Promise<MarketplacePluginList[]>;
```

Comment:
lists because its an array of PluginList:

```suggestion
  getPluginLists(): Promise<MarketplacePluginList[]>;
```